_posts/2020-10-27-best-ec2-instance-for-rails-apps.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "What's the Best EC2 Instance Type for Rails Apps?"
+date: 2020-10-27 10:00:00
+categories: ["rails", "performance", "ruby"]
+author: "noahgibbs"
+---
+
+<img src="/blog/assets/images/ec2_instance_pricing.png" style="max-width: 400px; float:right;" alt="EC2 us-east-1 on-demand instance prices - a huge grid of annoying numbers." />
```

Comment:
Can we put this image below the `<!--more-->` tag? so it only shows once you open the article and not in the preview in the index page

Also I'd remove the styles of it. otherwise it won't look good in small devices I think

Reply:
Can definitely move it down. I'm a little worried about removing the float:right because it kind of interrupts the article that way. I may just get rid of it in that case.

Reply:
Okay, got rid of it.